### PR TITLE
Upgrade to Kotlin 1.4

### LINKED ISSUE #6: Kotlin 1.4 release
Blocked on kotlinx.coroutines' 1.4 release

### PR DESCRIPTION
**Pull request by JakeWharton**
Closes #6.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -34,47 +34,30 @@ kotlin {
   sourceSets {
     commonMain {
       dependencies {
-        api 'org.jetbrains.kotlin:kotlin-stdlib-common'
```
Comment (Egorand) [inline build.gradle:37]: stdlib is added by default with 1.4, right?

File: build.gradle [hunk 2]
```diff
@@ -34,47 +34,30 @@ kotlin {
...
     }
     commonTest {
       dependencies {
-        implementation 'org.jetbrains.kotlin:kotlin-test-common'
-        implementation 'org.jetbrains.kotlin:kotlin-test-annotations-common'
+        implementation 'org.jetbrains.kotlin:kotlin-test-multiplatform'
       }
     }
 
     jsMain {
```
Comment (Egorand) [inline build.gradle]: Do we still need these empty blocks?

**Review by Egorand — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (JakeWharton) [reply build.gradle:37]: Yep

File: build.gradle (hunk 2 above)
Comment (JakeWharton) [reply build.gradle]: Not for these targets, no

**Pull request merged**